src/DynamoUtilities/DynamoFeatureFlagsManager.cs

```diff
@@ -87,12 +99,21 @@ internal T CheckFeatureFlag<T>(string featureFlagKey, T defaultval)
                 return defaultval;
             }
             // if we have not retrieved flags from the cli return empty
-            // and log.
+            // and log once,
            
             if(AllFlagsCache == null)
```

Comment:
is this thread safe ?

Reply:
no
![Screenshot 2023-11-22 at 1 01 59 PM](https://github.com/DynamoDS/Dynamo/assets/508936/76f6231f-43b7-4bf7-a8cd-12bb0f804121)